src/api/route.ts

```diff
@@ -0,0 +1,16 @@
+import axiosInstance from '@/api/api';
+
+type ChangeRouteReleased = {
+  routeId: number;
+  isPublic: boolean;
+  name?: string;
+};
+
+export const patchChangRouteReleased = async (routeReleased: ChangeRouteReleased) => {
+  try {
+    const response = await axiosInstance.patch(`/routes`, routeReleased);
+    if (response) return response.data;
```

Comment:
p5.
이부분을 이렇게 하는건 어떨까요?
```suggestion
    return response?.data;
```